.changeset/stale-wolves-type.md

```diff
@@ -0,0 +1,5 @@
+---
+"@comet/brevo-api": patch
+---
+
+Fix CSV import
```

Comment:
I would describe it a bit more.
Explaining that they type string[][] was wrong but seemed to worked earlier but with recent updated this lead to an error and you changed it to a JSONObject